README.md

```diff
@@ -5,3 +5,73 @@
 `legacy/vm_csv_parser.py` is the original program. However, it was not easy to test of for others to contribute. It is being kept around to ensure like-functionality in the new program `vminfo_parser`. VMInfo is a refactored (into classes) version with tests and a test dataset available for basic output testing.
 
 For more information on the original version, see its [README.md](legacy/README.md)
+
+## Installation
+
+vminfo_parser can be installed with pip, or executed directly from a cloned repo.
+
+### pip
+
+```
+pip3 install git+https://github.com/rhtools/vminfo-parser.git
+
+vminfo-parser --help
+```
+
+### git repo
+
+```
+git clone https://github.com/rhtools/vminfo-parser.git
+
+cd vminfo-parser
+pip3 install requirements.txt
+python3 -m vminfo_parser --help
+```
+
+## Usage
```

Comment:
Usage should also include a copy of the YAML or a reference to the files should it not?